scripts/start.py

```diff
@@ -132,11 +139,19 @@ def main(args=None):
     if not parsed_args.prod_env:
         # In prod mode webpack is launched through scripts/build.py
         python_utils.PRINT('Compiling webpack...')
-        background_processes.append(subprocess.Popen([
-            common.NODE_BIN_PATH,
-            os.path.join(
-                common.NODE_MODULES_PATH, 'webpack', 'bin', 'webpack.js'),
-            '--config', 'webpack.dev.config.ts', '--watch']))
+        if parsed_args.source_maps:
+            background_processes.append(subprocess.Popen([
+                common.NODE_BIN_PATH,
+                os.path.join(
+                    common.NODE_MODULES_PATH, 'webpack', 'bin', 'webpack.js'),
+                '--config', 'webpack.dev.sourcemap.config.ts', '--watch']))
+        else:
+            background_processes.append(subprocess.Popen([
+                common.NODE_BIN_PATH,
+                os.path.join(
+                    common.NODE_MODULES_PATH, 'webpack', 'bin', 'webpack.js'),
+                '--config', 'webpack.dev.config.ts', '--watch']))
```

Comment:
```suggestion
        webpack_config_file = (
            'webpack.dev.sourcemap.config.ts' if parsed_args.source_maps 
            else 'webpack.dev.config.ts')
        background_processes.append(subprocess.Popen([
            common.NODE_BIN_PATH,
            os.path.join(
                common.NODE_MODULES_PATH, 'webpack', 'bin', 'webpack.js'),
                '--config', webpack_config_file, '--watch']))
```
and have the file names be in constants.

Reply:
ah, nice! done!